web/html/src/manager/proxy/container-config.tsx

```diff
@@ -0,0 +1,401 @@
+              <input
+                name="proxyFQDN"
+                className="form-control"
+                type="text"
+                placeholder={t("e.g., proxy.domain.com")}
+                value={this.state.proxyFQDN}
+                onChange={this.proxyFQDNChanged}
+              />
+            </div>
+          </div>
+          <div className="form-group">
+            <label className="col-md-3 control-label">{t("Max Squid Cache Size (MB)")}:</label>
+            <div className="col-md-3">
+              <input
+                name="maxSquidCacheSize"
+                className="form-control numeric"
+                type="text"
+                placeholder={t("e.g., 2048")}
+                value={this.state.maxSquidCacheSize}
+                onChange={this.maxSquidCacheSizeChanged}
+                onKeyPress={window.numericValidate}
+              />
+            </div>
+          </div>
+          <div className="form-group">
+            <label className="col-md-3 control-label">{t("Proxy administrator email")}:</label>
+            <div className="col-md-6">
+              <input
+                name="proxyAdminEmail"
+                className="form-control"
+                type="text"
+                placeholder={t("e.g., proxy.admin@mycompany.com")}
+                value={this.state.email}
+                onChange={this.emailChanged}
+              />
+            </div>
+          </div>
+          <div className="form-group">
+            <label className="col-md-3 control-label">{t("SSL certificate")}:</label>
+            <div className="col-md-6">
+              <div className="radio col-md-3">
+                <input
+                  type="radio"
+                  name="ssl-mode"
+                  value="create-ssl"
+                  id="create-ssl"
+                  onChange={this.sslModeChanged}
+                  checked={this.state.sslMode === SSLMode.CreateSSL}
+                />
+                <label htmlFor="create-ssl" className="control-label">
+                  {t("Create")}
+                </label>
+              </div>
+              <div className="radio col-md-3">
+                <input
+                  type="radio"
+                  name="ssl-mode"
+                  value="use-ssl"
+                  id="use-ssl"
+                  onChange={this.sslModeChanged}
+                  checked={this.state.sslMode === SSLMode.UseSSL}
+                />
+                <label htmlFor="use-ssl" className="control-label">
+                  {t("Use existing")}
+                </label>
+              </div>
+            </div>
+          </div>
+          {this.state.sslMode === SSLMode.CreateSSL ? createSSLFields : useSSLFields}
+          <div className="form-group">
+            <br />
+            <div className="col-md-offset-3 col-md-6">{buttons}</div>
+          </div>
+        </div>
+      </TopPanel>
+    );
+  }
+}
+
+export const renderer = (id: string) => {
```

Comment:
In most other views we separate the renderer if I'm not mistaken, it helps keep files at a manageable size.